### ISSUE TITLE
Mugen?

### ISSUE THREAD

**Issue by Ahmed-Dzair-Dev** [kazzmir/paintown#102]
I see there is a mugen folder but i dont know how i can do to compile it to be come an excutable for linux ? Can u show me how ? 

I use only those ide " codeblocks or qt creator " i have other ide like : " Geany , VSC , ZeroBrane Studio ... "

**Comment by juvinious** [comment]
If you are on linux, which distribution are you on? 
If Ubuntu, clone the source and run `easy-compile-ubuntu` from the command line and it will pull all the requirements and build it for you. 

Read the [README.md](https://github.com/kazzmir/paintown/blob/master/README.md) text for instructions on building.

There are no real easy ways to do it from any of the IDE's you will need to do it from the command line.

**Comment by Ahmed-Dzair-Dev** [comment]
I want a standalone mugen without paintown is that possible for me ?

I have seen paintown have mugen but does mugen can be alone ?

Sorry for my bad english

**Comment by juvinious** [comment]
No, you can't build the mugen stuff separately. You can start into mugen directly from the executable using the appropriate flag `./paintown mugen` or `./paintown --mugen`.
